style.css

```diff
@@ -422,3 +433,215 @@ ul {
 .contact-form textarea {
   padding-top: 15px;
 }
+
+.portfolio-img-1-desktop {
+  display: none;
+}
+
+.text-desktop {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
```

Comment:
Good job for using the media query 👍 
 
**The issue that I have when I open your portfolio is that I have to scroll left and right to see the content.** 

Please update your design to be responsive by removing the scrolling.

<img width="904" alt="Capture d’écran 2022-06-03 à 00 20 42" src="https://user-images.githubusercontent.com/7243132/171746983-1b86c73b-fb66-4c23-a6f5-463f63d8afcb.png">